slices/bind9-host.yaml

```diff
@@ -0,0 +1,17 @@
+package: bind9-host
+
+essential:
+  - bind9-host_copyright
+
+slices:
+  bins:
+    essential:
+      - bind9-libs_libs
+      - libc6_libs
+      - libidn2-0_libs
+    contents:
+      /usr/bin/host:
+
+  copyright:
+    contents: 
```

Comment:
There appears to be a trailing space character at the end of this `contents:` line, can you remedy?